subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractGradleceptionSmokeTest.groovy

```diff
@@ -70,7 +70,7 @@ abstract class AbstractGradleceptionSmokeTest extends AbstractSmokeTest {
         def runner = testKitDir != null
             ? runnerWithTestKitDir(testKitDir, gradleArgs)
             : runner(*gradleArgs)
-        runner.ignoreDeprecationWarnings()
+        runner.ignoreDeprecationWarnings("Gradleception smoke tests don't check for deprecation warnings")
```

Comment:
I suppose the reason is https://github.com/gradle/gradle-private/issues/3405, so the deprecation warnings were flaky.

Reply:
Thanks, changed the comments.